skel/problem/generators/generators.yaml

```diff
@@ -60,3 +60,36 @@ data:
       #- large: generator {seed} -n 991
       #- large: generator {seed} -n 999
       #- large: generator {seed} -n 1000
+
+  invalid_inputs:
+    # Add invalid testcase to ensure that input_validators correctly rejects them.
+    data:
+      'empty':
```

Comment:
The testcase names don't require quotes, because the names by themselves are valid YAML keys. (This also holds for the commented-out `'dont_start_with_whitespace'`)